### DISCUSSION TITLE
Location page using TSML UI

### DISCUSSION THREAD

**Discussion by christpage** [code4recovery/12-step-meeting-list#1278] [Q&A] [answered]
Using the legacy UI I have a menu item pointing to a location page to show all of the meetings at a club as follows: https://aadaytononline.org/locations/alco-aides/

Is there a way to do this using the new TSML UI?

**Comment by SIA-WEBSUP** [comment]
I've wondered this myself. I'm not sure if TSML-UI supports 'location pages'. 

You could link to a search with the location name or the address to filter the meeting list to that location.
In your example: 
-     /?search=alco+aides+club  
-     /?search=2206+E+3rd+St

**Comment by joshreisner** [comment, marked as answer]
no, in TSML UI individual meeting screens display all the meetings at a location, so location screens are not needed

**Comment by jpiemeisl** [comment]
It would be helpful to be able to link to a page with all the meetings for a district. Is that possible?

**Reply by joshreisner** [reply]
sure, you could use search for this. you might want to use quotes around the search terms, example: https://sussexaa.org/meetings/?search=%22District+02%22
